libraries/standard/mqtt/integration-test/test_config.h

```diff
@@ -75,21 +64,36 @@
 #define BROKER_PORT    ( 8883 )
 
 /**
- * @brief Path of the file containing the server's root CA certificate.
+ * @brief Path of the file containing the client certificate.
  *
- * This certificate should be PEM-encoded.
+ * @note If using AWS IoT, refer to the AWS documentation below for details
+ * regarding client authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This certificate should be PEM-encoded.
+ *
+ * #define CLIENT_CERT_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path of the file containing the client's private key.
+ *
+ * @note If using AWS IoT, refer to the AWS documentation below for details
+ * regarding client authentication.
+ * https://docs.aws.amazon.com/iot/latest/developerguide/client-authentication.html
+ *
+ * @note This private key should be PEM-encoded.
+ *
+ * #define CLIENT_PRIVATE_KEY_PATH    "...insert here..."
  */
-#ifndef SERVER_ROOT_CA_CERT_PATH
-    #define SERVER_ROOT_CA_CERT_PATH    "certificates/mosquitto.org.crt"
-#endif
 
 /**
- * @brief MQTT client identifier.
+ * @brief Path of the file containing the server's root CA certificate.
  *
- * No two clients may use the same client identifier simultaneously.
+ * This certificate should be PEM-encoded.
  */
-#ifndef CLIENT_IDENTIFIER
-    #define CLIENT_IDENTIFIER    "testclient"
```

Comment:
Was this meant to be removed?

Reply:
Yes, as this config wasn't used by the integration test.